### DISCUSSION TITLE
Bot Won't Play Youtube Musuc and I don't know what to do.

### DISCUSSION THREAD

**Discussion by Dragontamer333** [jagrosh/MusicBot#1705] [Troubleshooting] [answered]
The bot no longer works with youtube and I'm not sure what to do to fix it. There is some referencing to logs and the like, but I don't know what "logs" are being mentioned. I am running the bot on a raspberry pi as a service. Anyone help me understand some steps to figuring what errors may get posted.

**Comment by MichailiK** [comment, marked as answer]
you can view the logs of any systemd service by using `journalctl -u servicename`, e.g. `journalctl -u JMusicBot`. You're likely experiencing the "Not success status code: 403" error though, which is already being tracked in #1682.

**Comment by Dragontamer333** [comment]
Ok thank you it is that error code.

**Comment by MWE001** [comment]
Same error I am getting as well as everyone else except it wont even let me play local files anymore.

**Comment by E-r1** [comment]
changing the lavaplayer version to 2.2.2 should fix it (https://github.com/jagrosh/MusicBot/blob/0.4.3/pom.xml then go to music dependencies)

**Reply by Dragontamer333** [reply]
I am trying to understand where to make this change. Everything is just running from the config and .jar file. Where do I find the pom.xml within my folders? 
![image](https://github.com/user-attachments/assets/ecf19966-8256-4260-a4d9-792488e452ea)

**Reply by polycountdev** [reply]
@Dragontamer333 You'll want to open your .jar file with winrar/some other archival tool and search/find 'pom.xml' in there. Make sure the filepath leads to the correct pom.xml. That said, E-r1's suggested fix did not work for me, even after changing lavaplayer's version number to 2.2.2.

**Comment by d-pacheco** [comment]
I have a fix, posted in this discussion: https://github.com/jagrosh/MusicBot/discussions/1708
